ProjectStructure.md

```diff
@@ -123,9 +172,98 @@ end
 
 - **TARGET MANAGEMENT**
 
-  This section has not been written yet.
+  Generally we have 3 targets which are *Production, Pre-Production, Staging*. According backend, we should create targets for each environment.  
+
+  For each target we have build flag like `-APPSTORE`, `-PREPROD`. If project has satellite apps, they should have custom flag.
+
+  To add a build flag, developer can add any build flag with opening a specific target's build settings and search "Preprocessor Macros" add build flag on necessary build configurations. 
+
+  To gather information about build flags on compile time, we created a `Environment.swift` file. 
+
+  	Sample **Environment.swift** 
```

Comment:
- Let's remove @objc tags.
- Let's keep only serverApi and serverHost variables as examples.
- It would be nice to add a variable for the analytics' credentials for an example.